### PR TITLE
[NCL-2527] Rewrited CLI tool from Bash to Python

### PR DESCRIPTION
**Pull request by stanlyDoge**
refactoring output is needed to do
to fully functionality needs to merge #349 and #347

### CODE REVIEW

File: cli-wrap/src/main/resources/cli/da-cli-script.py [hunk 1]
```diff
@@ -3,6 +3,8 @@
 import requests
 import sys
 
+#da_server = "pnc-da-cli.cloud.pnc.devel.engineering.redhat.com/da/rest/v-0.4"
+da_server = "localhost:8080/da/rest/v-0.4"
```
Comment (jomrazek) [inline cli-wrap/src/main/resources/cli/da-cli-script.py]: Don't know which instance is working, but this probably shouldn't be localhost

File: cli-wrap/src/main/resources/cli/testsuite.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 import sys
 import requests
 
-DA_MAIN_SERVER="ncl-test-vm-01.host.prod.eng.bos.redhat.com:8180/da/rest/v-0.4"
+DA_MAIN_SERVER="localhost:8080/da/rest/v-0.4"
```
Comment (jomrazek) [inline cli-wrap/src/main/resources/cli/testsuite.py]: probably same thing as above

Comment (stanlyDoge) [inline cli-wrap/src/main/resources/cli/testsuite.py]: ikr, I'm still working on formating. I'll replace localhost with some working instance at final push

Comment (jomrazek) [inline cli-wrap/src/main/resources/cli/testsuite.py]: ok, thanks

**Comment by stanlyDoge** [comment]
retest this please

**Comment by janinko** [comment]
add to help that python-requests is needed

**Comment by janinko** [comment]
I got:
```
$ ./da-cli.py list black
Traceback (most recent call last):
  File "./da-cli.py", line 378, in <module>
    CLITool()
  File "./da-cli.py", line 173, in __init__
    getattr(self, args.command.replace("-","_"))()
  File "./da-cli.py", line 212, in list
    getattr(self, "list_"+args.color.replace("-","_"))(parser)
  File "./da-cli.py", line 330, in list_black
    self.da.print_black_artifacts()
  File "/tmp/da-cli/da_cli_script.py", line 142, in print_black_artifacts
    for item in r.json():
  File "/usr/lib/python2.7/site-packages/requests/models.py", line 812, in json
    return complexjson.loads(self.text, **kwargs)
  File "/usr/lib64/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
```
when requesting ``http://pnc-da-cli.cloud.pnc.devel.engineering.redhat.com/da/rest/v-0.4/listings/blacklist`` that resulted with 404. Better error message would be appropriate when the server at ``da_server`` returns 404.

**Comment by janinko** [comment]
Command ``check white`` seems to partialy work, when it should not exist.

**Comment by janinko** [comment]
When there is mistake in a command, help for that command part should be printed.
e.g. something like:
```
$ ./da-cli.py list whitelist-ga org.jboss.as:jboss-as-jaxrs
Missing argument STATUS.
    python ./da-cli.py list whitelist-ga GROUP_ID:ARTIFACT_ID STATUS
        List all artifacts in the white list with a particular GA and status
        STATUS can be: SUPPORTED, SUPERSEDED, UNSUPPORTED, UNKNOWN
```

**Comment by stanlyDoge** [comment]
> Command check white seems to partialy work, when it should not exist.

I was just following old implementation. Should I remove this functionality?
https://github.com/project-ncl/dependency-analysis/blob/master/cli-wrap/src/main/resources/cli/listings.sh#L116

**Comment by janinko** [comment]
> >Command check white seems to partialy work, when it should not exist.
> I was just following old implementation. Should I remove this functionality?
> https://github.com/project-ncl/dependency-analysis/blob/master/cli-wrap/src/main/resources/cli/listings.sh#L116
Yes. It's probably remainder from time when the command existed.

**Comment by stanlyDoge** [comment]
retest this please

**Pull request merged**